apps/web-login/js/moudle/url_list.js

```diff
@@ -0,0 +1,9 @@
+let indexURL = new URL(document.location.href);
+let pathname = indexURL.pathname; // url-path 부분
+
+let callbackURL =
+  indexURL.origin + // scheme와 hosts 부분
+  pathname.substring(0, pathname.lastIndexOf("/")) +
+  "/CallBack.html";
+
+export { indexURL, callbackURL, pathname };
```

Comment:
이 부분은 상수가 아닌데 왜 따로 빼두었나요? 호출되는 부분을 확실히 알지 못하면 사이드이펙트가 발생할 가능성이 많습니다. 아니면 다음과 같이 함수로 빼둬서 호출 시점에 생성되도록 하는게 좋을 것 같네요.
```suggestion
function getIndexUrl() {
    return new URL(document.location.href)
}

function getCallbackUrl(indexUrl) {
    const origin = indexUrl.origin
    const pathname = indexUrl.pathname
    return origin + pathname.substring(0, pathname.lastIndexOf("/")) + "/CallBack.html"
}

export { getIndexUrl, getCallbackUrl };
```